library/photon

```diff
@@ -5,7 +5,11 @@ Maintainers: Fabio Rapposelli <fabio@vmware.com> (@frapposelli),
 GitRepo: https://github.com/vmware/photon-docker-image.git
 Directory: docker
 
-Tags: 2.0-20180302, 2.0, latest
+Tags: 2.0-20180409, 2.0, latest
+GitFetch: refs/heads/2.0-20180409
+GitCommit: 2d99288a999864936584df2d1774a18a7e5ec6e3
+
+Tags: 2.0-20180302
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

To put it another way, if there's an issue in `2.0-20180302`, will a `2.0-20180302.1` be released to fix it?  Or would that simply be a new snapshot tag?

Same with `1.0-XXX` -- is the `1.0` series still going to be maintained/supported?

Reply:
Thank you for taking care of it. Once I confirm with my team, I will update you. 
Thanks,
Michelle

Reply:
Hi @tianon,
We would like to keep all tags that were released. The reason is reproducible builds on customer side. They use specific date tags like `2.0-20180302` rather than just `2.0` or `latest`.
We are going to maintain 2 versions 1.0 and 2.0.

We are OK to release updates less frequently and to remove old/unused tags on demand.
Is there any way to track unused tags (number of downloads)?

Reply:
Any tag that has been built and pushed to the Docker Hub will stay indefinitely, regardless of what is in this `library/` file.

This file controls what shows up in the "Supported tags and respective Dockerfile links" section on the Docker Hub description and what is tried for builds. So from our build server standpoint they should usually not even be rebuilt since they are from scratch and the build context hasn't changed, but it does show up in "Supported Tags" and would imply to users that problems (like vulnerabilities) would be addressed in them.  But it sounds like these are point-in-time snapshots and will never be updated and as such, it seems like they should not be kept in the supported list once a newer snapshot is available (for example, `2.0-20180227` could be dropped when `2.0-20180302` is added).

(we can continue to discuss the old versions, but I'll go ahead and merge so that we can get the new images pushed)

Reply:
@tianon thank you for taking our attention to that problem. It was my misunderstanding of library file, especially regarding removed tags: "Tags that are removed from a library file do not get removed from the Docker Hub".
Currently, we maintain only 2 version of Photon OS. As result, library file has only 2 entries.